### ISSUE TITLE
Shell breaks Control Panel

### ISSUE THREAD

**Issue by MrNaughtZero** [moudey/Shell#575]
After installing this app, Control Panel breaks. The colors are different as seen in the screenshot below. Is this a known issue? Anyway to fix it?

I'm on Windows 11 24H2

![{A599947F-88F3-44EB-ABF2-FAA45A23AAD3}](https://github.com/user-attachments/assets/e54dfa84-1ffc-40e5-8fa0-a58702ae8378)

**Comment by DragonVsKira** [comment]
did this get fix?

**Comment by DragonVsKira** [comment]
did uninstall fix it?

**Comment by MrNaughtZero** [comment]
> did uninstall fix it?

Yes uninstalling fixes the issue.  I've decided to reinstall though. I very rarely use Control Panel, so it's not a huge issue for me.
